src/generators/genjs.ml

```diff
@@ -1912,7 +1917,7 @@ let generate com =
 	in
 	(* Provide console for environments that may not have it. *)
 	let closureArgs = if ctx.es_version < 5 then
-		var_console :: closureArgs
+		("console", typeof_join ["console"; "{log:function(){}}"]) :: closureArgs
```

Comment:
Does any js environments miss `console` implementation?

Reply:
In IE8/9 the console only works when `F12` is pressed